database/tables/assessment_questions.pg

```diff
@@ -31,6 +31,7 @@ columns
     max_submission_score_hist: double precision[]
     max_submission_score_variance: double precision
     mean_question_score: double precision
+    median_question_score: double precision
```

Comment:
One final note: you'll need to add this column to `AssessmentQuestionSchema` in `apps/prairielearn/src/lib/db-types.ts`.